src/services/AdvancedTradingView/adapters/advancedTradingView.adapter.ts

```diff
@@ -23,9 +26,9 @@ export abstract class AbstractAdvancedTradingViewAdapter<AppState> {
     this._adapterSupportedChains = adapterSupportedChains
   }
 
-  abstract getPairTrades(fetchDetails: AdapterFetchDetails): Promise<void>
+  abstract getPairTradingAndActivityData(fetchDetails: AdapterFetchDetails): any
 
-  abstract getPairActivity(fetchDetails: AdapterFetchDetails): Promise<void>
+  abstract getPairData(fetchDetails: AdapterFetchDetailsExtended): any
```

Comment:
This return promises, right? Maybe we can use `Promise<void>` type here

Reply:
This is resolved Leo.